doc/ref/cli/salt.rst

```diff
@@ -69,6 +69,20 @@ Options
     an explicit number of minions to execute at once, or a percentage of
     minions to execute on.
 
+.. option:: --batch-wait=BATCH_WAIT
+
+   Wait the specified time in seconds after each job is done before
+   freeing the slot in the batch of the next one.
+
+.. option:: --batch-safe-limit=BATCH_SAFE_LIMIT
+
+   Execute the salt job in batch mode if the job would have executed
+   on more than this many minions.
```

Comment:
In the code this is a >=, can we change this to:

"on this many minions or more"